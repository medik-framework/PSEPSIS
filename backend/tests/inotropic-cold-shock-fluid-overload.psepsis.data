{
  "BP Sys": 72,
  "MAP": 52,
  "Fluid Overload Signs": true
}
