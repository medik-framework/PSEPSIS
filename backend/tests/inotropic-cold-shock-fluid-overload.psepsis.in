[
  {
    "id": "tabletApp",
    "action": "broadcast",
    "eventName": "AgeEntered",
    "eventArgs": [60],
    "waitFor":
     {
       "name": "StartGUI"
     }
  },
  {
    "id": "tabletApp",
    "action": "broadcast",
    "eventName": "StartInotropicTherapy",
    "eventArgs": [],
    "waitFor" : {
      "name": "ConfirmAgeEntered"
    }
  },
  {
    "id": "tabletApp",
    "action": "broadcast",
    "eventName": "ConfirmBPObtained",
    "eventArgs": [],
    "waitFor" : {
      "name": "Instruct",
      "args": ["get BP"]
    }
  },
  {
    "id": "tabletApp",
    "action": "broadcast",
    "eventName": "ConfirmShockTypeMeasurementsObtained",
    "eventArgs": [],
    "waitFor" : {
      "name": "Instruct",
      "args": ["get shock type measurements"]
    }
  },
  {
    "id": "tabletApp",
    "action": "broadcast",
    "eventName": "ConfirmShockType",
    "eventArgs": ["Cold Shock"],
    "waitFor" : {
      "name": "Recommend",
      "args": [
        "get shock type"
        , "Fluid overload signs indicate decreased cardiac ability to deal with increased venous pressure. Consider using epinephrine to improve cadiac contractility."
        , "Recommendation: Cold Shock management"
      ]
    }
  },
  {
    "action": "exit",
    "waitFor" : {
      "name": "RecommendDrug"
    }
  }
]
