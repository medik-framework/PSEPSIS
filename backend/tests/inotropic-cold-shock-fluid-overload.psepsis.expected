[
  {
    "id": "tabletApp",
    "interface": "TabletApp",
    "name": "ConfirmAgeEntered",
    "args": []
  },
  {
    "id": "tabletApp",
    "interface": "TabletApp",
    "name": "Instruct",
    "args": ["get BP"]
  },
  {
    "id": "datastore",
    "interface": "Datastore",
    "name": "Obtain",
    "args": [
      "BP Sys"
    ]
  },
  {
    "id": "datastore",
    "interface": "Datastore",
    "name": "Obtain",
    "args": [
      "MAP"
    ]
  },
  {
    "id": "tabletApp",
    "interface": "TabletApp",
    "name": "Instruct",
    "args": ["get shock type measurements"]
  },
  {
    "id": "datastore",
    "interface": "Datastore",
    "name": "Obtain",
    "args": [
      "Fluid Overload Signs"
    ]
  },
  {
    "id": "tabletApp",
    "interface": "TabletApp",
    "name": "Recommend",
    "args": [
      "get shock type"
      , "Fluid overload signs indicate decreased cardiac ability to deal with increased venous pressure. Consider using epinephrine to improve cadiac contractility."
      , "Recommendation: Cold Shock management"
    ]
  },
  {
    "id": "tabletApp",
    "interface": "TabletApp",
    "name": "RecommendDrug",
    "args": [
      "Epinephrine",
      "Inotropes",
      "<1,10>Rat",
      "administer 0.1-1 mcg/kg/min IV/IO infusion"
    ]
  }
]
